src/cli/commands/bin.js

```diff
@@ -14,6 +14,6 @@ export function setFlags(commander: Object) {}
 
 export function run(config: Config, reporter: Reporter, flags: Object, args: Array<string>): Promise<void> {
   const binFolder = path.join(config.cwd, config.registries[RegistryYarn.registry].folder, '.bin');
-  reporter.log(binFolder);
+  reporter.log(binFolder, {force: true});
```

Comment:
I came across this [line](https://github.com/yarnpkg/yarn/blob/593e243364d28d4f106cd60b35a3d7de4ba37ae0/src/reporters/base-reporter.js#L200) but I can't think of a better solution atm :)

Reply:
I asked @arcanis to add that line 😃

Once we have some more time, we need to think about logging levels and warnings again.